### ISSUE TITLE
How do you configure PyCharm for this?

### ISSUE THREAD

**Issue by Laurentiu-Andronache** [pyscaffold/pyscaffold#223]
"python setup.py test" works, but how do I configure PyCharm for this?

Because just running pytest ends in an error where it can't import.

**Comment by FlorianWilhelm** [comment]
If you set up a proper virtual environment, run `python setup.py develop` after you activated it and configure that same environment in PyCharm it will run just fine.

**Comment by Laurentiu-Andronache** [comment]
So nice when pros help the peons! That solved it for me, thank you! 

I had only created the venv, then added the pre-commit hooks thingy without running "python setup.py develop". I was slightly depressed because of this issue, you have no idea.

So basically this is the workflow in order to develop:
$ cd to project directory
$ python -m venv venv
$ source .venv/bin/activate (or just venv\bin\activate on Windows)
$ pip install -e . (or python setup.py develop)
$ pip install pre-commit (because I ran putup with the "--pre-commit" argument)
$ pre-commit install
$ pre-commit autoupdate

Then start PyCharm, open the project directory, configure it to use the venv created. Enable PyTest from _Settings -> Tools -> Python Integrated Tools -> Testing_

Edit Configurations -> + -> Python tests -> pytest

Run the configuration created.

Then Ctrl+Alt+F6 in PyCharm -> + -> select .coverage file from the main project directory

Done!

On this last step, in the project tab is shows me coverage for irrelevant files as well, not just for those from the src/program folder. It shows me "0% files, not covered" for directories like venv and tests. Shouldn't those be ignored? Sorry if I'm talking non-sense.
